src/openforms/registrations/contrib/json_dump/tests/test_config.py

```diff
@@ -0,0 +1,10 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from ..config import validate_path
+
+
+class JSONDumpConfigTests(TestCase):
+    def test_validate_path(self):
+        with self.assertRaises(ValidationError):
+            validate_path("..")
```

Comment:
IMO this is too much implementation detail testing - instead test that the serializer blocks suspicious paths as that is our interface with/to the frontend/UI and even the registration tasks use the configuration serializers to create the options dict.